### ISSUE TITLE
Quick picker reading is broken with NVDA

### ISSUE THREAD

**Issue by Neurrone** [microsoft/vscode#93464]
Issue Type: <b>Bug</b>

1. Press ctrl+p and use the arrow keys to move through the list of items with NVDA running.

The entire contents of the window is read, instead of only the currently selected option.

VS Code version: Code - Insiders 1.44.0-insider (e62c4302b0e4f3e28bbcb0f3b6e46bb00e39679c, 2020-03-26T05:51:24.240Z)
OS version: Windows_NT x64 10.0.18363

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-7500U CPU @ 2.70GHz (4 x 2904)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off_ok<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.89GB (4.82GB free)|
|Process Argv||
|Screen Reader|yes|
|VM|0%|
</details><details><summary>Extensions (13)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-eslint|dba|2.1.1
gitlens|eam|10.2.1
prettier-vscode|esb|3.20.0
markdown-table-formatter|fcr|2.0.3
vscode-mysql|for|0.4.0
rest-client|hum|0.23.2
rust-analyzer|mat|0.2.100
Go|ms-|0.13.1
vscode-typescript-tslint-plugin|ms-|1.2.3
vsliveshare|ms-|1.0.1847
vscode-yaml|red|0.7.2
rust|rus|0.7.0
vscode-webhint|web|1.5.4

</details>

**Comment by bpasero** [comment]
I cannot reproduce this but if there is a regression we need to look into it:

![image](https://user-images.githubusercontent.com/900690/77726935-734a2400-6ff9-11ea-9d8e-af6bcdf60da0.png)

@Neurrone any more details what you did maybe?

**Comment by Neurrone** [comment]
Will let you know the next time I manage to reproduce this, it doesn't happen all the time.

**Comment by Neurrone** [comment]
Speak of the devil, I just reproduced this reliably with the latest insiders build.

1. Press `ctrl+shift+p` to open the commands list
2. Type `.` - in my case it says 795 results
3. I usually hear the first item being announced, but nothing is spoken at this point. Pressing down arrow then causes it to read all the items, not just the first.

**Comment by isidorn** [comment]
@bpasero you were using the role `list` for the quick pick. Which I think is wrong. I have changed now to use role `listbox` with role `option` for the items - same as suggest widget.
@Neurrone can you please try it out with VS Code insiders from Monday and let us know if this got fixed.

@marcozehe fyi since this is similar to #93516 
I believe these roles are correct for our quick pick widget - whcih basicaly lets the user choose one of the options and once that is done the widget goes away - thus a static role list does not work here.

**Comment by isidorn** [comment]
Similar issue which we fixed in the suggest widget by using listbox https://github.com/microsoft/vscode/issues/92719

**Comment by MarcoZehe** [comment]
Yes, this is correct. the "list" role is similar to an ul or ol element, a static list. "listbox" is what we want. And listbox only allows "option" children, so your fix is correct here as it was for the Suggest widget, too.

**Comment by isidorn** [comment]
@marcozehe Great, thanks. Let's also see how it behaves for @Neurrone from Monday.

**Comment by bpasero** [comment]
@isidorn thanks. Not however that I did not touch any of the ARIA things in quick input, that has always been there from @chrmarti work. Would be great if we could verify that we have not regressed while moving from the old classic tree widget to the list 👍

**Comment by isidorn** [comment]
@bpasero makes sure that in your QuickPick test plan item mention that people turn on Screen Reader and try it out. Also we havea bunch of screen reader users on insiders and they will also complain if something is off. As @Neurrone already complained.

**Comment by Neurrone** [comment]
Thanks for the quick fix! Yup, this is a duplicate of #93516
